docs/design/built-in-button-integration.md

```diff
@@ -0,0 +1,75 @@
+
+```xml
+<ExtensionPoint xsi:type="ContosoRibbonTab">
+  <CustomTab id="TabCustom1">
+    <Group id="myCustomTab.group1">
+       <!-- additional markup omitted -->
+    </Group>
+    <OfficeGroup groupId="Paragraph" />
+    <Label resid="customTabLabel1" />
+  </CustomTab>
+</ExtensionPoint>
+```
+
+## Insert a built-in control into a custom group
+
+To insert a built-in Office control into a custom group, add an [OfficeControl](../reference/manifest/group.md#officecontrol) element as a child element in the parent `<Group>` element. The `controlId` attribute of the of the `<OfficeControl>` element is set to the ID of the built-in control. See [Find the IDs of controls and control groups](#find-the-ids-of-controls-and-control-groups).
+
+The following is an example. This markup adds the Office Superscript control to a custom group and positions it to appear just after a custom button.
+
+```xml
+<ExtensionPoint xsi:type="ContosoRibbonTab">
+  <CustomTab id="TabCustom1">
+    <Group id="myCustomTab.grp1">
+        <Label resid="residCustomTabGroupLabel"/>
+        <Icon>
+            <bt:Image size="16" resid="blue-icon-16" />
+            <bt:Image size="32" resid="blue-icon-32" />
+            <bt:Image size="80" resid="blue-icon-80" />
+        </Icon>
+        <Control xsi:type="Button" id="Button2">
+            <!-- information on the control omitted -->
+        </Control>
+        <OfficeControl controlId="Superscript" />
+        <!-- other controls, as needed -->
+    </Group>
+    <Label resid="customTabLabel1" />
+  </CustomTab>
+</ExtensionPoint>
+```
+
+> [!NOTE]
+> Users can customize the ribbon in the Office application. Any user customizations will override your manifest settings. For example, a user can remove a button from any group and remove any group from a tab.
+
+## Find the IDs of controls and control groups
+
+[!INCLUDE [publish policies note](../includes/find-control-ids.md)]
+
+## Behavior on unsupported platforms
+
+If your add-in is installed on a platform that does not support [requirement set AddinCommands 1.3](../reference/requirement-sets/add-in-commands-requirement-sets.md)), then the markup described in this article is ignored and the built-in Office controls/groups will not appear in your custom groups/tabs. To prevent your add-in from being installed on platforms that don't support the markup, add a reference to the requirement set to the `<Requirements>` section of the manifest. For instructions, see [Set the Requirements element in the manifest](../develop/specify-office-hosts-and-api-requirements.md#set-the-requirements-element-in-the-manifest). Note that the runtime checks that are described in the article are not applicable to **AddinCommands 1.3** because it contains only manifest markup that is read when the add-in starts up and before a runtime check can occur.
```

Comment:
```suggestion
If your add-in is installed on a platform that does not support [requirement set AddinCommands 1.3](../reference/requirement-sets/add-in-commands-requirement-sets.md), then the markup described in this article is ignored and the built-in Office controls/groups will not appear in your custom groups/tabs. To prevent your add-in from being installed on platforms that don't support the markup, add a reference to the requirement set in the `<Requirements>` section of the manifest. For instructions, see [Set the Requirements element in the manifest](../develop/specify-office-hosts-and-api-requirements.md#set-the-requirements-element-in-the-manifest). Note that the runtime checks that are described in the article are not applicable to **AddinCommands 1.3** because it contains only manifest markup that is read when the add-in starts up and before a runtime check can occur.
```